uktrade/data-hub-frontend
# Hide upcoming details summary for archived (cancelled) interactions

## Change
Does not show the details summary for upcoming interactions when the interaction is archived (cancelled).

## Test instructions
When using the `DEV` api:
1. Browse to `~/companies/0fb3379c-341c-4da4-b825-bf8d47b26baa/activity-feed`
2. Browse to the details of a `Cancelled interaction`
3. You should no longer see the `Why can I not complete this interaction?` details summary

## Screenshot
![Screenshot 2019-06-20 at 08 23 54](https://user-images.githubusercontent.com/1150417/59829245-cf7a6c00-9334-11e9-979d-e407debe09a0.png)

**Checklist**
